### PR TITLE
Cron backup, Adminer and Docker README updates

### PR DESCRIPTION
**Pull request by robbyoconnor**
This revamped the section on setting up Librehealth EHR with a new db

### CODE REVIEW

**Comment by robbyoconnor** [comment]
@tmccormi I included screenshots where I felt it was needed. This should be good to go. Let me know if you want me to refactor or clarify. 

One thing we need to address is that this probably should be behind HTTPS but stated that this is out of the scope, we should add support for this at some point.

**Comment by robbyoconnor** [comment]
Going to add one more screenshot of the final step of setup

**Comment by robbyoconnor** [comment]
Okay -- I think I'm done -- let me know if I missed anything

**Comment by robbyoconnor** [comment]
The screenshots don't show in the rich diff so look here: https://github.com/robbyoconnor/lh-ehr/tree/docker-readme-fixes/docker

**Comment by robbyoconnor** [comment]
Added Adminer to both dev and prod :)

File: docker-compose.prod.yml [hunk 1]
```diff
@@ -1,23 +1,31 @@
...
+      - "8000:80"
+    restart: always
+    env_file: ./docker/.env.ehr
+    depends_on:
+      - db
+  adminer:
+    image: adminer:4
+    restart: always
+    ports:
+      - "8080:8080"
```
Comment (tmccormi) [inline docker-compose.prod.yml]: 8080 is typically the port Java TomCat runs under (think Jenkins) maybe a 8010 or something?

Comment (robbyoconnor) [reply docker-compose.prod.yml]: what about 8001?

Comment (robbyoconnor) [reply docker-compose.prod.yml]: Perfect :)

**Comment by robbyoconnor** [comment]
Okay -- I have some slight fixes -- I added automated backup based on OpenMRS' cron backup but tweaked it and made some improvements to it and pushed it to https://gitlab.com/librehealth/lsc/lh-docker-cron-backup/container_registry via `registry.gitlab.com/librehealth/lsc/lh-docker-cron-backup:latest` 

Have a peek and see if it works.

**Comment by robbyoconnor** [comment]
Okay should be good now.

**Comment by robbyoconnor** [comment]
I did a lot more work -- so you'll want to have a peek at it...I think I was clear -- I tried to make no assumptions.

**Comment by robbyoconnor** [comment]
@tmccormi  Added the Bash scripts for managing things

**Comment by robbyoconnor** [comment]
I am going to merge 8546105292f81e3f8d69f43b7da017b6e40fe3aa  (which was the last approved revision and create a new pull request with the rest.

**Pull request closed without merging**